hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -352,12 +352,12 @@ private void handleOperationError(Operation operation, Throwable e) {
 
         if (operation instanceof Backup) {
             failedBackupsCounter.inc();
-        }
-
-        if (!operation.returnsResponse()) {
             return;
         }
 
+        // a response is send regardless of the Operation.returnsResponse method because some operation do want to send
+        // back a response, but they didn't want to send it yet but they ran into some kind of error. If on the receiving
```

Comment:
is it logged (no invocation found...)as a warning? or just debug?

Reply:
logger.warning("No Invocation found for normal response with callId " + callId + " sent from " + sender);

Reply:
so this change can generate quite some noise?

Reply:
perhaps we could use `finest` when `callId == 0`? (=no invocation ever existed?)